lib/core/src/msParam.cpp

```diff
@@ -1022,6 +1022,34 @@ parseMspForPosInt( msParam_t* inpParam ) {
     return myInt;
 }
 
+int msp_to_string(msParam_t* inp_param, char** out_param)
+{
+    if (out_param == nullptr) {
+        log_msi::error("{}: out_param is null", __func__);
+        return SYS_NULL_INPUT;
+    }
+
+    if (inp_param == nullptr || inp_param->inOutStruct == nullptr) {
+        log_msi::error("{}: inp_param is null or _inpParam's inOutStruct is null", __func__);
+        *out_param = nullptr;
+        return SYS_NULL_INPUT;
+    }
+
+    if (strcmp(inp_param->type, STR_MS_T) != 0) {
+        log_msi::error("{}: inp_param type {} is not STR_MS_T", __func__, inp_param->type);
```

Comment:
Let's wrap the input argument's placeholder in square brackets. That way, if we receive an empty string, it's noticeable.

```
"{}: inp_param type [{}] is not STR_MS_T"
```